exasol_advanced_analytics_framework/lua/src/event_loop.lua

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
What do you think, can we integrate this module also in other repositories to build custom event loop scripts with maybe custom parameters?